CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Release 2.7.0 (in development)
 ### Enhancements
+- Replace the prometheus-operator chart by the kube-prometheus-stack one and
+  bump the version to 12.2.3
+  (PR [#2948](https://github.com/scality/metalk8s/pull/2948))
```

Comment:
Would be great to see which version of components are deployed with this chart (e.g. Grafana 7.2.1)... Maybe we can automate something, once we work on changelog from fragments.